src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -83,55 +86,44 @@ public static object ConvertFromJson(string input, bool returnHashTable, out Err
                     });
 
                 // JObject is a IDictionary
-                var dictionary = obj as JObject;
-                if (dictionary != null)
+                if (obj is JObject dictionary)
                 {
-                    if (returnHashTable)
-                    {
-                        obj = PopulateHashTableFromJDictionary(dictionary, out error);
-                    }
-                    else
-                    {
-                        obj = PopulateFromJDictionary(dictionary, out error);
-                    }
+                    obj = returnHashTable ?
+                              PopulateHashTableFromJDictionary(dictionary, out error) :
+                              PopulateFromJDictionary(dictionary, memberTracker, out error);
```

Comment:
When reaching here, we already know the size of the `JObject`, so maybe we can replace `memberTracker` with `new DuplicateMemberSet(dictionary.Count)`? (just like what you did in other places)
Then `DefaultMemberSetCapacity` won't be needed.